docs: add glossary page (#15)
CERNDocumentServer/docs-cds-rdm: docs/glossary.md

# Glossary

## Record lifecycle

Metadata

> :   Descriptive information about a record, such as title, authors, resource type, and identifiers. Metadata can be edited after a record is published; files cannot.

Author: add resource type. and cross link

Draft

> :   A record that has been saved but not yet published. Drafts are only visible to the uploader and anyone explicitly granted access. They do not appear in search results.

Reviewer: ```suggestion
:   A record that has been saved but not yet published. Keep in mind that published does not mean public! Drafts are only visible to the uploader and anyone explicitly granted access. They do not appear in search results.
```
Please explain the public (restriction) vs published (not draft) in easy to understand way

Published record

> :   A record that has been finalised and registered in the system. Once published, files can no longer be changed. A published record is not necessarily public — its visibility depends on the access settings configured before publishing.

Author: ```suggestion
:   A record that has been finalised and registered in the system. Once published, files can no longer be changed. A published record is not necessarily public — its visibility depends on the access settings configured.
```

> :   A record that has been finalised and registered in the system. Once published, files can no longer be changed. A published record is not necessarily public — its visibility depends on the access settings configured before publishing.

Reviewer: the part about it can't be changed is not entirely true - we can modify metadata but not files. I think it needs to be phrased clearer

New version

> :   A draft copy created from a published record, used when files need to be modified or replaced. All versions remain accessible and are linked together in the record's version history.

Author: ```suggestion
:   A draft created from a published record, used when files need to be modified or replaced. All versions remain accessible and are linked together in the record's version history.
```

and cross link to draft/published record etc

Embargo
:   A time-based restriction applied to one or more files. Embargoed files are not accessible until the specified date, even if the record's metadata is public.

> ## Visibility & access

Author: add small text, to learn more see access&share

Public
:   A record or file that is visible to anyone, including unauthenticated users.

Restricted
:   A record or file that requires a CERN account or explicit permission to access. A restricted record may have public metadata while its files remain restricted.

Access request

> :   A request submitted by an unauthorised user to gain access to restricted files. The record owner or designated approver receives a notification to approve or deny the request. Access requests can be enabled or disabled from the Share interface.

Author: ```suggestion
:   A request submitted by a user to gain access to restricted files. The record owner receives a notification to approve or deny the request. Access requests can be enabled or disabled using the Share button.
```

> :   A request submitted by an unauthorised user to gain access to restricted files. The record owner or designated approver receives a notification to approve or deny the request. Access requests can be enabled or disabled from the Share interface.

Reviewer: ```suggestion
:   A request submitted by an unauthorised user to gain access to restricted files or record. The record owner or designated approver receives a notification to approve or deny the request. Access requests can be enabled or disabled from the Share interface.
```

Shareable link

> :   A link that grants access to a draft or record to users who cannot log in to CDS. Shareable links can be generated from the Share interface.

Author: ```suggestion
:   A link that grants access to a draft or record to users who cannot log in to CDS. Shareable links can be generated using the Share button.
```

## Identifiers

DOI

> :   A Digital Object Identifier — a persistent identifier registered with [DataCite](https://datacite.org) when a record is published. CDS can register a DOI automatically, or you can provide an existing external DOI.

Author: ```suggestion
:   A Digital Object Identifier — a persistent identifier when a record is published. CDS can register a DOI automatically, or you can provide an existing external DOI.
```

add doi doc reference

Report number (RN)

> :   A CERN-specific identifier for a record. Auto-assigned in legacy CDS.

Author: ```suggestion
:   A CERN-specific unique identifier for a record.
```

> :   A CERN-specific identifier for a record. Auto-assigned in legacy CDS.

Reviewer: ```suggestion
:   An alternative identifier for a record which tends to be used in research. It should not be treated aa unique identifier
```
Report numbers are not CERN specific, used also in other institutions

> ## Community & roles

Author: add a ref to community doc

Community

> :   Collection of records managed by a group of users, typically representing an experiment, department, or project at CERN. A record can belong to more than one community.

Author: mention record has to belong at least one community

Curator
:   A community member with permission to review submission requests, edit records that belong to the community, and manage the community's content.

Inclusion request

> :   A request to add an already published record to an additional community. Since a record can belong to more than one community, an inclusion request allows it to be curated by another community without creating a new version or going through the submission process.

Author: ```suggestion
:   A request to add an already published record to an additional community. Since a record can belong to more than one community, an inclusion request allows it to be curated by another community without creating a new version.
```

Submission request

> :   A request to include a record in a community. Submitting a record to a community triggers a review by the community's curators.

Author: ```suggestion
:   A request to publish in a community. Submitting a record sends it to the community’s curators for review..
```

Review

> :   The process by which a community curator evaluates a submission request and either accepts or declines the record's inclusion in the community.

Author: ```suggestion
:   The process where a curator evaluates a submission and determines whether it should be accepted into the community.
```

#### Public

A record or file that is visible to anyone, including unauthenticated users.

#### Restricted

A record or file that requires a CERN account or explicit permission to access. A restricted record may have public metadata while its files remain restricted.

#### Access request

A request submitted by a user to gain access to restricted files. The record owner receives a notification to approve or deny the request. Access requests can be enabled or disabled using the Share button.

#### Shareable link

A link that grants access to a [draft](#draft) or record to users who cannot log in to CDS. Shareable links can be generated using the Share button.

## Identifiers

#### DOI

A Digital Object Identifier — a persistent identifier when a record is published. CDS can register a DOI automatically, or you can provide an existing external DOI. See [what is a DOI?](https://www.doi.org/the-identifier/what-is-a-doi/) for a general introduction.

#### Report number (RN)

A CERN-specific unique identifier for a record.

## Community & roles

To learn more, see [About communities](../communities/communities.md).

#### Community

> Collection of records managed by a group of users, typically representing an experiment, department, or project at CERN. A record must belong to at least one community and may belong to more than one.

Reviewer: Please rephrase it to avoid using word "collection", we are about to introduce collection as a feature and there will be name clash.
Also community is more about curation and management of records (so grouping people with specific rights), not about grouping records together
